### ISSUE TITLE
'A mod sens' volume bug

### ISSUE THREAD

**Issue by Sentinel77** [asb2m10/dexed#66]
Hi,

 First of all,  great app. I'm really enjoying it. Thanks for your great work on this, and everyone else involved. It's amazing how close it is to the real hardware.

 I found a bug affecting the volume. It occurs with patches that have Amplitude Modulation Sensitivity turned up.
The volume of dexed (0.9.1) can be about 8.8dB lower with those patches, whenever you open your project or open a new instance of dexed.
The volume returns to normal if you touch your midi controller keyboards modulation wheel,
or if you open the 'PARM' window and hit 'OK'.

example:
- open a new instance of dexed.
- hit 'INIT' to create a new sound.
- turn up 'A mod sens' on operator 1.
  the volume will sound rather low.
- hit 'PARM' and press 'OK',  Or  make some movement on the modulation wheel of the midi controller keyboard,  and the volume will now be about 8.8 dB louder.

thanks.

**Comment by MD-78** [comment]
Confirmed (0.9.1 x64).

**Comment by asb2m10** [comment]
Thanks for the feedback and reporting this. Will try to fix this in 0.9.2

**Comment by asb2m10** [comment]
Hi @Sentinel77 , I've added the fix you suggested plus removed the reset of the eg_bias if the values were not 0; I need to check out why I did that before saying it is fixed.

**Comment by Sentinel77** [comment]
Hi @asb2m10 

 I tested it, but it didn't work. - couldn't turn off Eg Bias.

Your code that resets eg_mod  looks ok to me  --  when all Eg Bias switches are 'Off' then it sets Eg Bias modulation to 127.  This looks correct, so maybe it's ok to keep this.


 I tested out some more ideas and I found something that is working.  https://github.com/Sentinel77/dexed/commit/b50f5d334fb71dec2d3b697da03ff9a5db3f4106
When you get a chance take a look and see if it's ok, or if there's an improved way to do it.

Thank you.
